<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IBSSS_Stream_View_Window_Layout</class>
 <widget class="QMainWindow" name="IBSSS_Stream_View_Window_Layout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="changeImageButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Image</string>
      </property>
     </widget>
     <widget class="Stream_Display" name="Display_1" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>251</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="Stream_Display" name="Display_2" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="Stream_Display" name="Display_3" native="true">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="Stream_Display" name="Display_4" native="true">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>290</y>
        <width>251</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Streams</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="streamLink_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>211</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Paste Stream link below:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="streamLink">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PIN">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>181</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="PIN_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>51</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>PIN:</string>
      </property>
     </widget>
     <widget class="QLabel" name="addStreams_label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add Streams</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="back1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>480</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>381</width>
        <height>411</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>jasjsadjl;adsf</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ajl;asdf;jladsf</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="ManageStreams_label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Manage Streams</string>
      </property>
     </widget>
     <widget class="QListWidget" name="userList">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="ManageUsers_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Manage Users</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="back3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>89</x>
        <y>79</y>
        <width>491</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="oldPW_label">
           <property name="text">
            <string>Old password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="old_pw"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="newPW_label">
           <property name="text">
            <string>New password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="new_pw"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="confirmPW_label">
           <property name="text">
            <string>Confirm password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="confirm_pw"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="changePasswordButton">
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAdd_Stream"/>
    <addaction name="actionManage_Streams"/>
    <addaction name="actionManage_Users"/>
    <addaction name="actionChange_Password"/>
    <addaction name="separator"/>
    <addaction name="actionLog_out"/>
   </widget>
   <addaction name="settingsMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManage_Streams">
   <property name="text">
    <string>Manage Streams</string>
   </property>
  </action>
  <action name="actionLog_out">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionAdd_Stream">
   <property name="text">
    <string>Add Stream</string>
   </property>
  </action>
  <action name="actionManage_Users">
   <property name="text">
    <string>Manage Users</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Stream_Display</class>
   <extends>QWidget</extends>
   <header>ibsss_stream_display.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
